src/include/miopen/sqlite_db.hpp

```diff
@@ -278,38 +282,50 @@ class SQLiteBase
         return AllFound;
     }
     template <typename... U>
-    inline auto FindRecord(U&... args)
+    inline boost::optional<DbRecord> FindRecord(U&... args)
     {
+        if(DisableDbFileIO)
```

Comment:
This change disables both layers of SQLite-based system perf-db, which is `MultiFileDb<SQLitePerfDb, SQLitePerfDb, true>`, because `SQLitePerfDb` inherits from this class.

We shall keep the SQLite-based system perf-db in working state (or it is disabled in the baseline `develop`???)